tests/notebook-tests.cfg

```diff
@@ -7,9 +7,9 @@ regex: id="\d*"
 replace: ID-STAMP
 
 [regex3]
-regex: \/\S+\/scmdata\/src\S+\:
+regex: \/\S+\/scmdata\/src\S+\:.*:?
```

Comment:
What does this do?

Reply:
makes sure that line number differences don't cause things to fail (no idea why there are line number differences, but anyway...